fastlane/Fastfile

```diff
@@ -96,18 +96,36 @@ def distribute_builds(
       extension: 'app.zip',
       name: 'Mac Universal'
     },
+    mac_universal_dmg: {
+      binary_path: File.join(BUILDS_FOLDER, 'Studio-darwin-universal', 'Studio.dmg'),
+      filename_core: 'darwin-universal',
+      extension: 'dmg',
+      name: 'Mac Universal (DMG)'
+    },
```

Comment:
Glad the DRY from #117 turned out to be useful. I wasn't expecting us to add new files to upload, but there you go. Happy it was here.